cli/report/report.py

```diff
@@ -15,24 +15,37 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 import fnmatch
+import json
 import os
 import shutil
 from datetime import datetime
 from typing import Any
 from typing import Optional
 
 import jira
+from jira import Issue
 from simple_logger.logger import get_logger
 
+from cli.gitleaks import get_default_gitleaks_config
+from cli.gitleaks.gitleaks import GitleaksConfig
 from cli.objects.configuration import Configuration
 from cli.objects.failure import Failure
 from cli.objects.failure_rule import FailureRule
+from cli.objects.gitleaks_detection import GitleaksDetection
+from cli.objects.gitleaks_detection import GitleaksDetectionCollection
+from cli.objects.gitleaks_detection import GitleaksDetectionsJobFailure
 from cli.objects.jira_base import Jira
 from cli.objects.job import Job
+from cli.objects.rule import Rule
 
 
 class Report:
-    def __init__(self, firewatch_config: Configuration, job: Job) -> None:
+    def __init__(
+        self,
+        firewatch_config: Configuration,
+        job: Job,
+        gitleaks: bool = False,
```

Comment:
I think it may make more sense to make the `gitleaks` value be an item in the configuration.[ Here is an example](https://github.com/CSPI-QE/firewatch/blob/main/cli/objects/configuration.py#L60). That way we don't need to pass it around, we can just set it in the `firewatc_config` and check the config during execution.

Reply:
I agree, that would make more sense. I'll work on moving that over.